packages/back-end/src/init/event-emitters.ts

```diff
@@ -0,0 +1,42 @@
+import { getEventEmitterInstance } from "../services/event-emitter";
+import {
+  APP_NOTIFICATION_EVENT_EMITTER_NAME,
+  NotificationEventName,
+  NotificationEventPayload,
+  NotificationEventResource,
+} from "../events/base-types";
+import { eventLoggingHandler } from "../events/handlers/event-logging/eventLoggingHandler";
+
+let initialized = false;
+
+export const initializeEventEmitters = () => {
+  if (initialized) {
+    console.warn("Not re-initializing event emitters.");
+    return;
+  }
+
+  const eventEmitter = getEventEmitterInstance();
+
+  eventEmitter.on(
+    APP_NOTIFICATION_EVENT_EMITTER_NAME,
+    (
+      event: NotificationEventPayload<
+        NotificationEventName,
+        NotificationEventResource,
+        unknown
+      >
+    ) => {
+      console.log(
+        "EventEmitter -> emitted:",
+        APP_NOTIFICATION_EVENT_EMITTER_NAME
+      );
+
+      eventLoggingHandler(event);
+      // slackEventHandler(event);
+      // webHooksEventHandler(event);
```

Comment:
This is the one handler enabled so far. The webHooks one will be enabled in the next set of changes. `eventLoggingHandler` logs the events to the database.